docs/cloud/cloud-databases/cloud-db-serverless-scale.md

```diff
@@ -0,0 +1,51 @@
+---
+title: Scale a serverless database
+layout: default
+parent: Manage databases
+grand_parent: Cloud
+nav_order: 3
+---
+
+# How do I scale a serverless up or down database?
```

Comment:
The original heading has a typo and doesn't make sense.

Scale implies direction, so you don't need "up or down"

Therefore I suggest the following:

```
# How do I scale a Serverless Database?
```